pages/api/gifts/index.ts

```diff
@@ -34,11 +31,46 @@ export default async function handler(
 }
 
 async function handleGET(req: NextApiRequest, res: NextApiResponse) {
-  const giftRequest = await axios.get(`${baseURL}`);
-  return res.status(giftRequest.status).json(giftRequest.data as Gift[]);
+  const gifts = await prisma.gift.findMany({
```

Comment:
Lisäämällä tyypin `NextApiResponse`en välttää virheitä, kun TypeScript huomauttaa jos paluutyyppi on väärä. Tämä ei aiemmin toiminut, kun Axios ei palauttanut tyyppejä, mutta nyt Prisman kanssa toimii hienosti. Laittaisin sen jokaiseen API-funktioon joka palauttaa dataa.
```suggestion
async function handleGET(req: NextApiRequest, res: NextApiResponse<Gift[]>) {
```